cattle/plugins/host_info/disk.py

```diff
@@ -0,0 +1,44 @@
+import platform
+
+from cattle.utils import CadvisorAPIClient
+from cattle import Config
+
+
+class DiskCollector(object):
+    def __init__(self):
+        self.unit = 1024.0
+        self.cadvisor = CadvisorAPIClient(Config.cadvisor_ip(),
+                                          Config.cadvisor_port())
+
+    def _convert_units(self, number):
+        return round(float(number)/self.unit, 3)
+
+    def _get_mountpoints_cadvisor(self):
+        data = {}
+        stat = self.cadvisor.get_latest_stat()
+
+        if 'filesystem' in stat.keys():
+            for fs in stat['filesystem']:
+                device = fs['device']
+                percent = float(fs['usage']) / float(fs['capacity']) * 100
+
+                data[device] = {
+                    'free': self._convert_units(fs['capacity'] - fs['usage']),
+                    'total': self._convert_units(fs['capacity']),
+                    'used': self._convert_units(fs['usage']),
+                    'percent': round(percent, 2)
```

Comment:
Percent what? Used or free? Do you have the power to change this or is it expected to be just "percent" on the cattle side?